roles/kvirt_vm/tasks/main.yml

```diff
@@ -0,0 +1,7 @@
+---
+- name: Run validations
+  ansible.builtin.include_tasks: validations.yml
+
```

Comment:
I think the deletion should happen here when forced

Reply:
sure, done